### PR TITLE
[ENH] Update 2.4.0 + new features nextflow

### PR DESCRIPTION
**Pull request by arnaudbore**

### CODE REVIEW

**Comment by arnaudbore** [comment]
Fix #13

File: source/installation/install.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ Download the last release of TractoFlow pipeline:
 
 ::
 
-    $> wget https://github.com/scilus/tractoflow/releases/download/2.2.1/tractoflow-2.2.1.zip && unzip tractoflow-2.2.1.zip
+    $> wget https://github.com/scilus/tractoflow/releases/download/2.3.0/tractoflow-2.3.0.zip && unzip tractoflow-2.3.0.zip
```
Comment (GuillaumeTh) [inline source/installation/install.rst]: If you say to use -r in the launch page, you need to install with the nextflow pull

Comment (arnaudbore) [reply source/installation/install.rst]: Fixed

File: source/pipeline/launch.rst [hunk 2]
```diff
@@ -9,14 +9,14 @@ To run the pipeline, use the following command:
 ::
 
     # With Singularity
-    $> nextflow run tractoflow/main.nf --bids input_bids --dti_shells "DTI_SHELLS" --fodf_shells "FODF_SHELLS" -with-singularity singularity_name.img -resume
+    $> nextflow run tractoflow -r 2.4.0 --bids input_bids --dti_shells "DTI_SHELLS" --fodf_shells "FODF_SHELLS" -with-singularity singularity_name.sif -resume
     # Or
-    $> nextflow run tractoflow/main.nf --root input_folder --dti_shells "DTI_SHELLS" --fodf_shells "FODF_SHELLS" -with-singularity singularity_name.img -resume
+    $> nextflow run tractoflow -r 2.4.0 --root input_folder --dti_shells "DTI_SHELLS" --fodf_shells "FODF_SHELLS" -with-singularity singularity_name.sif -resume
```
Comment (CHrlS98) [inline source/pipeline/launch.rst]: `--root` is now `--input` no?

File: source/reference/changelog.rst [hunk 3]
```diff
@@ -1,6 +1,26 @@
 Changelog
 =========
 
+2.4.0
+########
+Date: October 2022
+
+New features
```
Comment (CHrlS98) [inline source/reference/changelog.rst:8]: Nothing?

**Review by CHrlS98 — COMMENTED** [review]
Just the small comments!

**Review by CHrlS98 — APPROVED** [review]

**Pull request merged**